test_server.py

```diff
@@ -3,7 +3,9 @@
 #import server as app
 import requests
 
-
+def test_demo():
+    assert 1==1
+"""
```

Comment:
are you trying to comment?

Reply:
Yes, I commented the present tests, as they are outdated and we also had to run `server.py` for the tests, so I will rewrite the tests after `travis` is configured properly.

Reply:
Let's use actual comments instead of multiline strings? Shows up better in a diff too.

Reply:
So, I will have to comment all lines...

Reply:
whatever editor you are using will have support for block commenting. Actually forget this. No point in enforcing this. Anyways this is going to go out right?

Reply:
Done. Yes it will go when I rewrite the tests.